public/components/preview_panel/index.tsx

```diff
@@ -56,33 +63,45 @@ export const PreviewPanel = ({ onClose, model }: Props) => {
 
   const respondingStatus = useMemo(() => {
     if (loading) {
-      return (
-        <EuiTextColor color="subdued" data-test-subj="preview-panel-color-loading-text">
-          Loading...
-        </EuiTextColor>
-      );
+      return {
+        overall: (
+          <EuiTextColor color="subdued" data-test-subj="preview-panel-color-loading-text">
+            Loading...
+          </EuiTextColor>
+        ),
+        nodes: 'Loading...',
+      };
     }
     const deployedNodesNum = nodes.filter(({ deployed }) => deployed).length;
     const targetNodesNum = nodes.length;
     if (deployedNodesNum === 0) {
-      return (
-        <EuiHealth className="ml-modelStatusCell" color="danger">
-          Not responding on {targetNodesNum} of {targetNodesNum} nodes
-        </EuiHealth>
-      );
+      return {
+        overall: (
+          <EuiHealth className="ml-modelStatusCell" color="danger">
+            Not responding
+          </EuiHealth>
+        ),
+        nodes: `Not responding on ${targetNodesNum} of ${targetNodesNum} nodes`,
```

Comment:
Naming: prefer use `nodesStatusText` for all. Could you confirm with UX about if we need to use `Not responding on` here?